config/deploy.rb

```diff
@@ -12,3 +12,6 @@
 
 append :linked_files, '.env.production', 'public/robots.txt'
 append :linked_dirs, 'vendor/bundle', 'node_modules', 'public/system'
+
+set :assets_prefix, 'packs'
```

Comment:
Will this work? Because `assets:precompile` executes both Sprockets and Webpack, there are still parts that use Sprockets, such as the Sidekiq and PGHero web views, so those things remain under "assets" I believe, while Webpack stuff goes into "packs".